Admin/MenuNodeAdmin.php

```diff
@@ -160,10 +160,10 @@ public function getFormBuilder()
      */
     public function buildBreadcrumbs($action, MenuItemInterface $menu = null)
     {
-        $menuNodeNode = parent::buildBreadcrumbs($action, $menu);
+        $menuNodeItem = parent::buildBreadcrumbs($action, $menu);
```

Comment:
its a MenuItem, not a MenuNode. This wrong naming has me confused in 2 PRs now, so let's change it. :)